docs/release/release_0_2_4.rst

```diff
@@ -0,0 +1,50 @@
+Announcement: napari 0.2.4
+==========================
+
+We're happy to announce the release of napari v0.2.4!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-base
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+New Features
+------------
+- Add gamma slider (#610)
+
+Improvements
+------------
+- Make _on_data_change only get called once when ndisplay changes (#629)
+- fix undefined variables, remove unused imports (#633)
+- Raise informative warning when no Qt loop is detected (#642)
+
+API Changes
+-----------
+
```

Comment:
`napari.Viewer.add_multichannel` was removed. Use `napari.Viewer.add_image(..., channel_axis=num)` instead.

Reply:
Oh yeah, this is an API change, not a deprecation, which means, something is now discouraged and will be removed in a future version.

Reply:
changed